README.md

```diff
@@ -6,6 +6,8 @@ This is the frontend for the **MyPokedex** project, a React-based app that conne
 
 For the backend API, check out the [MyPokedex-Backend](https://github.com/iameddysmith/MyPokedex-Backend).
```

Comment:
You `backend` repo contains `node_modules`, which is too huge for downloading. Please, remove it from `github` 
![image](https://github.com/user-attachments/assets/d9a2f05c-6603-4020-af15-25f1d5b37a7b)

Reply:
I have took time off to build this project and have been working tirelessly on this project. I cleared out the node_modules from the back end. I pushed all my current updates to my back end as well as this pull request.